pkg/addon/common.go

```diff
@@ -35,7 +35,6 @@ type GlobalValues struct {
 	ImagePullPolicy string            `json:"imagePullPolicy,"`
 	ImagePullSecret string            `json:"imagePullSecret"`
 	ImageOverrides  map[string]string `json:"imageOverrides,"`
-	NodeSelector    map[string]string `json:"nodeSelector,"`
```

Comment:
Does this remove NodeSelector from being configured by annotation?

Reply:
I don't think we had a test for it via annotation, so if that isn't a required behavior, that's fine.

Reply:
I tried the following example and it did overwrite the tolerations on the deployment:
```yaml
apiVersion: addon.open-cluster-management.io/v1alpha1
kind: ManagedClusterAddOn
metadata:
  name: config-policy-controller
  annotations:
    addon.open-cluster-management.io/values: '{"tolerations": [{"key": "dedicated", "operator": "Equal", "value": "something-else", "effect": "NoSchedule"}]}'
spec:
  installNamespace: open-cluster-management-agent-addon
```